fishjaw/model/model.py

```diff
@@ -450,6 +464,7 @@ def predict(
     :param patch_size: the size of the patches to use
     :param patch_overlap: the overlap between patches. Uses a hann window
     :param activation: the activation function to use
+    :param batch_size: the batch size to use
```

Comment:
id consider changing this to something like "maximum number of patches that will be simultaneously sent to the model", because usually we only talk about batch size when were training